spec/models/profile_spec.rb

```diff
@@ -135,14 +135,19 @@
       expect {@profile.image_url = ""}.not_to change(@profile, :image_url)
     end
 
-    it 'makes relative urls absolute' do
-      @profile.image_url = "/relative/url"
-      expect(@profile.image_url).to eq("#{@pod_url}/relative/url")
+    it 'makes absolute urls relative' do
+      @profile.image_url = "https://host/bar/relative/url"
+      expect(@profile.image_url).to eq("/bar/relative/url")
+    end
```

Comment:
This change however, is incorrect. Those profile fields are federated to other pods and used as they are there. Thus making the URLs relative breaks the profile pictures on remote pods.